features/actions.md

```diff
@@ -0,0 +1,31 @@
+# Claims
+
+Claims are the main way to create new tokens for your community. We use the concept of Objectives and Actions.
+
+### Objectives
+
+Objectives are what you want to achieve with your actions. All actions must be done towards a bigger objective. We recommend giving a through description to your objectives and try to add an expected result for it: a **good** result and a **great** result. This will help you get a better grasp on how your expectations are aligned with the reality.
+
+### Actions
+
+Actions are the definition of exactly what the community must do to reach those objectives. We made actions in a way that they are very flexible and allow for different use cases. For that we added a few options:
+
+**_Validity_ of an action. This can be by _date_ and _quantity_**
+  - _Date_: its pretty straightforward. The action can be claimed until a certain date. After that date is met, its no longer possible to open new claims. An important part is that the claims themselves can be validated after this date is met. Only new claims that cannot be created.
```

Comment:
Here is my suggestion for this section: 
- _Date_: An action can be claimed until certain date is reached. After the set date, claims cannot be opened. The validation of opened claims can occur after the set date. The opening of new claims cannot occur after the set date.

Reply:
What you think of removing the repetition?

```suggestion
  - _Date_: An action can be claimed until certain date is reached. After the set date, claims cannot be opened. The validation of opened claims can occur after the set date.
```

Reply:
Good idea!